doc/guides/security-release-process.md

```diff
@@ -2,118 +2,88 @@
+    * v10.x, v12.x: ***LINK to PR URL***
+  * ...
+
+* [ ] PR release announcements in [private](https://github.com/nodejs-private/nodejs.org-private):
+  * (Use previous PRs as templates, don't forget to update the site banner, and
+    the date in the slug so that it will move to the top of the blog list.)
+  * [ ] pre-release: ***LINK TO PR***
+  * [ ] post-release: ***LINK TO PR***
 
 * [ ] Get agreement on the planned date for the release: ***RELEASE DATE***
 
-* [ ] Validate that all vulnerabilities have been assigned a CVE. Upstream deps
-  such as OpenSSL and NPM will have CVEs, issues reported on H1 may have CVEs,
-  otherwise allocate them by following the
-  [cve_management_process](https://github.com/nodejs/node/blob/master/doc/guides/cve_management_process.md).
+* [ ] Get release team volunteers for all affected lines:
+  * v12.x: ***NAME of RELEASER(S)***
+  * ... other lines, if multiple releasers
 
-* [ ] Co-ordinate with the Release team members to line up one or more releasers
-  to do the releases on the agreed date. Releaser: ***NAME of RELEASER(S)***
+## Announcement (one week in advance of the planned release)
 
-* [ ] Prep for the security announcements by getting agreement on drafts (use
-  previously announced releases as the template):
-  * pre-release: ***LINK TO COMMENT ON THIS ISSUE CONTAINING DRAFT***
-  * post-release: ***LINK TO COMMENT ON THIS ISSUE CONTAINING DRAFT***
+* [ ] Check that all vulnerabilities are ready for release integration:
+  * PRs against all affected release lines or cherry-pick clean
+  * Approved
+  * Pass `make test`
+  * Have CVEs
+  * Described in the pre/post announcements
 
-## Announcement (one week in advance of the planned release)
+* [ ] Pre-release announcement [email][]: ***LINK TO EMAIL***
+  (Get access from existing manager: Ben Noordhuis, Rod Vagg, Michael Dawson)
 
-* [ ] Send pre-release announcement to
-  https://groups.google.com/forum/#!forum/nodejs-sec.
-  One of the existing managers can give access (Ben
-  Noordhuis, Rod Vagg, Michael Dawson). ***LINK TO EMAIL***
-
-* [ ] Post pre-release announcement in vulnerabilities section of Nodejs.org
-  blog (https://github.com/nodejs/nodejs.org/tree/master/locale/en/blog/vulnerability).
-  Use last pre-release announcement as a template (it includes blog metadata
-  such as updates to the banner on the Node.js website to indicate security
-  releases are coming).  Submit PR and land immediately. Text was already
-  reviewed in security repo.  ***LINK TO BLOG PR AND POST***
-
-* [ ] Open an issue in the build working repository with a notification of the
-  date for the security release.  Use this issue to co-ordinate with the build
-  team to ensure there will be coverage/availability of build team resources the
-  day of the release. Those who volunteer from the build WG should be available
-  in node/build during the release in case they are needed by the individual
-  doing the release. ***LINK TO BUILD ISSUE***
```

Comment:
Still not sure we have consensus to drop this (notification to build WG) but I'm not going to block it.

Reply:
A month in review and no objections! Still, I pushed this back into the text:
```
* [ ] Notify build-wg of upcoming security release date by opening an issue
  in [nodejs/build][] to request WG members are available to fix any CI issues.
```